### PR TITLE
Quickswap: bugs and errors

### PR DESCRIPTION
**Pull request by NotEternal**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [x] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [x] I tested desktop/mobile resolution
- [x] I tested light/dark theme
- [x] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [ ] I checked the work on the Testnet
- [x] I checked the work on the Mainnet
- [ ] I checked the work in the plugin
- [x] I checked the **PR** once again

## Tests

Please start auto tests as follows:

- add a label <ins>swap test</ins> to start swap tests
- add a label <ins>withdraw test</ins> to start withdraw tests

You can skip these tests if you completely sure that your changes aren't related to this functional

### Original issue
#4743
#4746

### Video / screenshot proof

image

### CODE REVIEW

File: src/front/shared/pages/Exchange/QuickSwap/index.tsx [hunk 1]
```diff
@@ -774,8 +807,17 @@ class QuickSwap extends PureComponent<IUniversalObj, ComponentState> {
       <>
         <NewTokenInstruction />
 
+        {receivedCurrency.notExist && (
+          <p styleName="noAssetsNotice">
+            <FormattedMessage
+              id="notEnoughAssetsNotice"
+              defaultMessage="You do not have enough assets in the wallet. Please add more of them to be able to swap."
```
Comment (noxonsu) [inline src/front/shared/pages/Exchange/QuickSwap/index.tsx:814]: You don't have available assets in this network to exchange. Please change the network or add a custom asset to the wallet

**Comment by developer24-exe** [comment]
builded https://testnet4750.swaponline.site ;

**Comment by developer24-exe** [comment]
commit 8741db3f1 rebuilded https://testnet4750.swaponline.site https://mainnet4750.swaponline.site

**Review by noxonsu — APPROVED** [review]

**Comment by developer24-exe** [comment]
commit 1844c3841 rebuilded https://testnet4750.swaponline.site https://mainnet4750.swaponline.site

**Comment by developer24-exe** [comment]
commit 248d958b2 rebuilded https://testnet4750.swaponline.site https://mainnet4750.swaponline.site

**Pull request merged**